tools/ci_build/github/pai/pai_test_launcher.sh

```diff
@@ -13,3 +13,6 @@ echo ""
 
 echo "Running ./onnxruntime_test_all .."
 $build_dir/onnxruntime_test_all --gtest_filter=$gtest_filter
+
+echo "Running ./onnxruntime_provider_test .."
+$build_dir/onnxruntime_provider_test --gtest_filter=$gtest_filter
```

Comment:
this file is not referenced in any other repo code. can it be removed?

Reply:
Yes.